src/database/queries/userExtension.js

```diff
@@ -245,7 +247,7 @@ module.exports = class MenteeExtensionQueries {
 			}
 
 			let projectionClause =
-				'user_id,meta,visibility,organization_id,designation,area_of_expertise,education_qualification'
```

Comment:
@adithyadinesh0412 Should we change the visibility to mentor_visibility anyway you have added the new key to projection. But do not remove existing projection, it may cause logical issues